catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -401,10 +400,6 @@ public static void setAutoCrashReportingEnabled(Context context, boolean isEnabl
 		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
 		editor.putBoolean(SETTINGS_CRASH_REPORTS, isEnabled);
 		editor.commit();
-
-		if (isEnabled) {
-			CrashReporter.initialize(context);
-		}
```

Comment:
Thanks for letting me know that the testrun will be red without these lines of code :smile: